packages/toolchain-go/collection.yaml

```diff
@@ -68,12 +68,18 @@ packages:
   - name: toolchain-go
     category: fips
     variant: "alpine"
-    version: "1.19.10"
-    tag: "1.19.10"
+    version: "1.22.5"
```

Comment:
why we are moving alpine from 1.19 to 1.22 but on bullseye we are adding an extra package?

Reply:
we are adding an extra package in both cases. But in the diff it look like we are "moving" the alpine one, because in parallel, I'm removing the duplicate entry for 1.19.10 :D.

Reply:
it's clearer if you use the "split" view